app/src/main/java/com/blogspot/e_kanivets/moneytracker/activity/record/AddRecordActivity.kt

```diff
@@ -103,6 +106,14 @@ class AddRecordActivity : BaseBackActivity() {
         tvDate.setOnClickListener { selectDate() }
         tvTime.setOnClickListener { selectTime() }
 
+        if (type == Record.TYPE_EXPENSE) {
+            fabDone.backgroundTintList = (ColorStateList.valueOf(ContextCompat.getColor(this, R.color.red_light)))
```

Comment:
`ColorStateList.valueOf(ContextCompat.getColor(this,` is duplicated, so presumably can be extracted.